cpp/src/Slice/Parser.h

```diff
@@ -453,7 +453,6 @@ namespace Slice
         EnumeratorList enumerators() const;
         EnumeratorList enumerators(const std::string& identifier) const;
         ContainedList contents() const;
-        std::string thisScope() const;
```

Comment:
This function is only used internally.
Instead of keeping it public and adding the `mappedName` flag, I just made it protected, so only things in `libSlice` (which don't care about mapped names) can access it.